lib/rc_networking/src/server/mod.rs

```diff
@@ -113,7 +115,7 @@ impl ServerSocket {
                     user.write_packets.send(SendPacket(
                         Protocol::Disconnect(Disconnect::new(0)),
                         UserId(0),
-                    ));
+                    )).expect("failed to send disconnect packet");
```

Comment:
This was purposefully left without an unwrap because if the users socket has already shut and the tokio threads despawned, then the reciever for this is gone and it will fail. We don't really care if it failed to send a disconnect, so it shouldn't panic the server if it did fail

Reply:
The proper way to discard a Result is something like
`let _discard = `
which will not trigger the unused Result compiler lint.
I will make a commit to update this.